docs/bootstrap.md

```diff
@@ -1,35 +1,56 @@
-# Bootstrapping a Kubernetes cluster
+# Bootstrapping Dragonfly on Digital Ocean
 
-## Create namespaces
+## Prerequisites
 
-```bash
-kubectl apply -f kubernetes\manifests\cert-manager\namespace.yaml
-kubectl apply -f kubernetes\manifests\discord\namespace.yaml
-kubectl apply -f kubernetes\manifests\dragonfly\namespace.yaml
+You will need:
+
+* [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/) configured with a...
+* [Digital Ocean Personal Access Token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) with at least the following scopes:
+  * kubernetes (2): create, delete
+  * If replacing an existing cluster:
+    * load\_balancer (1): delete
+* PWD set to the root of this repo
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [`helm`](https://helm.sh/docs/intro/install/)
+
+## Create the cluster in Digital Ocean
+```
+doctl k8s cluster create \
+  <cluster-name> \
+  --size s-2vcpu-2gb \
+  --count 2 \
+  --region nyc3
 ```
 
-## Install the Helm Chart to get all the dependencies
+Adjust `size` and `count` as needed. See [`doctl kubernetes cluster create
+docs`](https://docs.digitalocean.com/reference/doctl/reference/kubernetes/cluster/create/)
+for more options.
+
+## Apply `cert-manager` CRDs
 
-```bash
-helm install -f kubernetes\chart\production.yaml vipyrsec kubernetes\chart\
+```
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.crds.yaml
 ```
 
-## Apply the Discord bot deployment
+## Install the Helm Chart for `ingress-nginx` and `cert-manager`
 
 ```bash
-kubectl apply -f kubernetes\manifests\discord\bot
+helm upgrade vipyrsec ./kubernetes/chart --install
 ```
 
-## Apply the Dragonfly Mainframe deployment
+## Apply manifests and secrets
+
+Secrets are not included in this repo; you will need to create your own.

```

Comment:
`suggestion(blocking)`: Moving the manifests to the Helm chart will solve this, but for now, it is what it is.

```suggestion
You might need to run the below command twice, since the namespaces might not get created before the other manifests depending on them.
Secrets are not included in this repo, so you will need to create those separately.
```